gpytorch/settings.py

```diff
@@ -681,6 +681,40 @@ class skip_logdet_forward(_feature_flag):
     _default = False
 
 
+class _linalg_symeig(_value_context):
```

Comment:
Suggest calling this `_linalg_dtype_symeig` (and `_linalg_dtype_*` more generally) for clarity. g